### PR TITLE
Min-height avoids scrolling. Scrolling happens because cell text is t…

### PR DESCRIPTION
**Pull request by DiegoAlbertoTorres**
…oo short

### CODE REVIEW

**Comment by samccone** [comment]
Can you post before and after screenshots? and how this fixes things in a non brittle way?

**Comment by scottdraves** [comment]
this adds too much space below the code box.

File: core/src/main/web/app/styles/cells.scss [hunk 1]
```diff
@@ -23,6 +23,12 @@
     padding-top: $grid-row-height;
 }
 
+.CodeMirror-scroll {
+    overflow-y: hidden !important;
```
Comment (samccone) [inline core/src/main/web/app/styles/cells.scss]: can we put this into vendor overrides and also drop !important

**Comment by samccone** [comment]
screen shot 2015-06-11 at 9 38 31 am
screen shot 2015-06-11 at 9 38 14 am

:boat: :+1:

**Pull request merged**